feat: add name parameter to Envied annotation

techouse opened the pull request:
This addresses https://github.com/petercinibulk/envied/issues/2 and adds the option to [change the generated class name just like envify does](https://github.com/frencojobs/envify#change-generated-class-name).

Additionally, I fixed a unit test by adding a missing environment variable to the `.env.example`.

```.env
SYSTEM_VAR=system_var
```

The above environment variable was missing, resulting in the `envied_generator` test for `Env10` failing.

Because of that test fix, I did not squash both commits into a single commit.

petercinibulk commented:
@techouse  This looks like a great PR. I am always willing to add more flexibility. Can you fix the errors in the CI and then I will get it merged and released?

techouse commented:
Heh, my bad. For some reason, IntelliJ didn't enable IntelliSense inside the example directory. So naturally, I missed the errors. I fixed all the examples and added an even more verbose example of an `AppEnv`.
